src/intervals.js

```diff
@@ -0,0 +1,30 @@
+import "./App.css";
+
+const Intervals = () => {
+  return (
+    <>
+      <div className="padding-container" id="intervals-section">
+        <div className="container-flex">
+          <div className="title">Intervals</div>
+          <div className="align-center">
+            <p>Whole tone - W</p>
+            <p>Half tone - H</p>
+          </div>
+        </div>
+        <div className="separation-line"></div>
```

Comment:
Could use an hr tag and style it here, I think. Or maybe use borders on the previous or following elements to create a separation line.